orchestrator/workflow.py

```diff
@@ -212,6 +214,11 @@ def wrapping_function() -> NoReturn:
     wrapping_function.target = target
     wrapping_function.steps = steps
 
+    wf_table = get_workflow_by_name(wrapping_function.name)
```

Comment:
As discussed, it's safer to perform this DB query in the "Start process" code than here, to avoid a stale `.is_task` value in memory